docs/_quickstart/recommended-triggers-and-actions.md

```diff
@@ -572,4 +572,4 @@ View [example apps](https://zapier.com/apps/categories/webinars).
 {% endcapture %}
 {{ styled_table | markdownify | replace: '<table>', '<table class="even-columns">' }}
 
-View [example apps](https://zapier.com/apps/categories/cms).
\ No newline at end of file
+View [example apps](https://zapier.com/apps/categories/cms).
```

Comment:
Adding a newline at the end of the file. Otherwise, just renamed the file from must-have-triggers-and-actions to recommended-triggers-and-actions.